tools/scxctl/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[[bin]]
+name = "scxctl"
+
+[package]
+name = "scxctl"
+version = "1.0.9"
+edition = "2021"
+description = "A cli dbus client for scx_loader"
+authors = ["Joe Maples <joe@maples.dev>"]
+repository = "https://github.com/sched-ext/scx"
+readme = "README.md"
+keywords = ["linux", "scx", "scx_scheds", "scx_loader", "sched_ext"]
+categories = ["command-line-utilities"]
+license = "GPL-2.0-only"
+
+[profile.release]
```

Comment:
I think this gets picked up by the top level cargo, so this is probably not needed.